util/chplenv/chpl_comm_substrate.py

```diff
@@ -15,7 +15,6 @@ def get():
     if not substrate_val:
         comm_val = chpl_comm.get()
         platform_val = chpl_platform.get('target')
-        arch_val = chpl_arch.get('target', get_lcd=True)
```

Comment:
Why is this here?

Reply:
Oh, nevermind. I see it's dead code  (that I missed removing in https://github.com/chapel-lang/chapel/commit/29ec9bce3d2b30cd15d84c2d0d07e8c3b806f5d0#diff-45c544a18723ce3d378babf320393a0fL28)